components/bl-double-calendar-component/component.json

```diff
@@ -0,0 +1,143 @@
+      "name": "specificDateRange",
+      "label": "Specific date range",
+      "settingTooltip": "This handler allows you to select the date selection mode in a range.",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "specificDateRangeLogic",
+      "handlerLabel": "Specific date range logic",
+      "handlerDescription": "This handler allows you to select the date selection mode in a range.",
+      "dataBinding": true
+    },
+    {
+      "type": "number",
+      "name": "selectionRange",
+      "label": "Selection range",
+      "settingTooltip": "This handler allows you to specify a date selection range. This will work if you select a 'Specific date range' mode.",
+      "showInSettings": true,
+      "defaultValue": "7",
+      "hasLogicHandler": true,
+      "handlerId": "selectionRangeLogic",
+      "handlerLabel": "Selection range logic",
+      "handlerDescription": "This handler allows you to specify a date selection range. This will work if you select a 'Specific date range' mode.",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "selectedDate",
+      "label": "Selected date",
+      "settingTooltip": "This handler allows you to specify the default selected date. Date format to be specified \"year/month/day\".",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "selectedDateLogic",
+      "handlerLabel": "Selected date logic",
+      "handlerDescription": "This handler allows you to specify the default selected date. Date format to be specified \"year/month/day\".",
+      "dataBinding": true
+    },
+    {
+      "type": "checkbox",
+      "name": "displayHeader",
+      "label": "Display header",
+      "settingTooltip": "This handler allows you to control the display of the header.",
+      "showInSettings": true,
+      "defaultValue": true,
+      "hasLogicHandler": true,
+      "handlerId": "displayHeaderLogic",
+      "handlerLabel": "Display header logic",
+      "handlerDescription": "This handler allows you to control the display of the header."
+    },
+    {
+      "type": "text",
+      "name": "dateFrom",
+      "label": "Date from",
+      "settingTooltip": "This handler allows you to specify a start date. Date format to be specified \"year/month/day\".",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "dateFromLogic",
+      "handlerLabel": "Date from logic",
+      "handlerDescription": "This handler allows you to specify a start date. Date format to be specified \"year/month/day\".",
+      "dataBinding": true
+    },
+    {
+      "type": "text",
+      "name": "dateTo",
+      "label": "Date to",
+      "settingTooltip": "This handler allows you to specify an end date. Date format to be specified \"year/month/day\".",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "dateToLogic",
+      "handlerLabel": "Date to logic",
```

Comment:
Not sure, need to ask somebody with good grammar, but may be will be better to write

`From Date Logic` and `To Date Logic`